producer.go

```diff
@@ -226,22 +215,16 @@ stop:
 				log.Println("Received message to send. Total messages received: " + strconv.FormatInt(p.getMsgCount(), 10))
 			}
 
-			kargs := p.args()
-			fargs := p.firehoseArgs()
-
-			if p.getProducerType() == kinesisType {
-				kargs.AddRecord(msg.Value(), string(msg.Key()))
-			} else if p.getProducerType() == firehoseType {
-				fargs.AddRecord(msg.Value(), string(msg.Key()))
-			}
+			// kargs := p.args()
+			// fargs := p.firehoseArgs()
```

Comment:
can we remove or perhaps add a conditional that will enable us to use these sorts of arguments?